sklearn/ensemble/_forest.py

```diff
@@ -884,9 +884,9 @@ class RandomForestClassifier(ForestClassifier):
     A random forest is a meta estimator that fits a number of decision tree
     classifiers on various sub-samples of the dataset and uses averaging to
     improve the predictive accuracy and control over-fitting.
-    The sub-sample size is always the same as the original
-    input sample size but the samples are drawn with replacement if
-    `bootstrap=True` (default).
+    The sub-sample size is controlled with the max_samples parameter if
+    bootstrap=True (default), otherwise the whole dataset is used to build each
+    tree.
```

Comment:
```suggestion
    The sub-sample size is controlled with the `max_samples` parameter if
    `bootstrap=True` (default), otherwise the whole dataset is used to build each
    tree.
```

Reply:
Thanks! I copy-pasted the rendered text instead of the markdown source...